### PR TITLE
Fixes #33553 -  Allow "Enter" key for submit action on all CV forms

### PR DESCRIPTION
**Pull request by Andrewgdewar**
### What are the changes introduced in this pull request?
This adds the "Enter" key submit option in 3 locations within CV UI: 
- Component Content View Add modal
- Copy Content View Modal
- Create Content View Form

### What are the testing steps for this pull request?
- Navigate to the above mentioned modals and "Tab" to the fields required for validation. 
- Without selection the Submit/Save/Update button, hit "Enter" to fire the submit action and close the modal.

### CODE REVIEW

File: webpack/scenes/ContentViews/Copy/CopyContentViewForm.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import useDeepCompareEffect from 'use-deep-compare-effect';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import { Redirect } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Copy/CopyContentViewForm.js:7]: 谢谢 🥳

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Copy/CopyContentViewForm.js:7]: :)

**Review by sjha4 — APPROVED** [review]
Submit with enter key now works on all 3 forms..
APJ 👍🏽

**Comment by sjha4** [comment]
[test katello]

**Pull request merged**